src/frontend/packages/lib_components/src/utils/createContext.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { createContext, useContext, useState } from 'react';
+import React, { createContext, useContext, useEffect, useState } from 'react';
```

Comment:
in the commit description

`s/was not updating his value/was not updating its value`